spec/discourse_akismet/user_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rails_helper'
+
+describe DiscourseAkismet::User do
+  before do
+    SiteSetting.akismet_api_key = 'not_a_real_key'
+    SiteSetting.akismet_enabled = true
+  end
+
+  let(:user) { Fabricate(:newuser) }
+
+  random_bio = 'random profile'
```

Comment:
Variable outside test methods should be declared using this syntax

```suggestion
  let(:random_bio) { 'random profile' }
```